credentials/tls/certprovider/meshca/plugin_test.go

```diff
@@ -46,9 +46,12 @@ import (
 )
 
 const (
-	defaultTestCertLife = time.Hour
-	shortTestCertLife   = 2 * time.Second
-	maxErrCount         = 2
+	defaultTestTimeout           = 100 * time.Millisecond
```

Comment:
As mentioned earlier just now: feel free to make these much longer to reduce flakiness. The goal is to avoid wasted time when we are waiting for something to NOT happen.  Not to set tight bounds and detect a problem where something SHOULD happen quickly.  For whatever reason, I like something around 5-10s -- it's tons of time for a lagging travis instance, but not that long to wait before detecting a bug.  (If you are iterating on a bug you might temporarily change it to be even shorter though.)